thunder/core/jit_ext.py

```diff
@@ -743,6 +744,42 @@ def grad_transform(*args, **kwargs):
     return forward_result
 
 
+@register_general_jit_lookaside(torch.Tensor._make_wrapper_subclass)
```

Comment:
Does it need to be a jit lookaside? Can the implementation be moved to `thunder/torch/__init__.py` if `@torchsymbol` is used?

Reply:
I rather want to hide this method as possible. so making it a lookaside feels more right than a torchsymbol